src/esn.calendar.libs/app/components/event/form/event-form.controller.js

```diff
@@ -217,6 +220,15 @@ function CalEventFormController(
     }
   }
 
+  function isValidURL(string) {
+    if (!string) {
+      return;
+    }
+    const res = string.match(/(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g);
+
+    return (res !== null);
```

Comment:
```suggestion
    return string.match(/(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g);
```

Reply:
the test of null must be present , your suggestion break the interface